source/frontend/src/features/mapSideBar/acquisition/tabs/expropriation/form1/ExpropriationForm1.tsx

```diff
@@ -0,0 +1,110 @@
+import { SectionField } from '@/components/common/Section/SectionField';
+import { RestrictContactType } from '@/components/contact/ContactManagerView/ContactFilterComponent/ContactFilterComponent';
+import FilePropertiesTable from '@/components/filePropertiesTable/FilePropertiesTable';
+import { Api_AcquisitionFile } from '@/models/api/AcquisitionFile';
+import { Api_PropertyFile } from '@/models/api/PropertyFile';
+
+import { ExpropriationForm1Model } from '../models';
+import { ExpropriationForm1YupSchema } from './ExpropriationForm1YupSchema';
+
+export interface IExpropriationForm1Props {
+  acquisitionFile: Api_AcquisitionFile;
+  onGenerate: (values: ExpropriationForm1Model) => Promise<void>;
+}
+
+export const ExpropriationForm1: React.FC<IExpropriationForm1Props> = ({
+  acquisitionFile,
+  onGenerate,
+}) => {
+  const onSubmit = async (
+    values: ExpropriationForm1Model,
+    formikHelpers: FormikHelpers<ExpropriationForm1Model>,
+  ) => {
+    try {
+      await onGenerate(values);
+    } finally {
+      formikHelpers?.setSubmitting(false);
+    }
+  };
+
+  const onCancelClick = (formikProps: FormikProps<ExpropriationForm1Model>) => {
+    formikProps.resetForm();
+  };
+
+  const onGenerateClick = (formikProps: FormikProps<ExpropriationForm1Model>) => {
+    formikProps.setSubmitting(true);
+    formikProps.submitForm();
+  };
+
+  return (
+    <Formik<ExpropriationForm1Model>
+      enableReinitialize
+      initialValues={new ExpropriationForm1Model()}
+      validationSchema={ExpropriationForm1YupSchema}
+      onSubmit={onSubmit}
+    >
+      {formikProps => (
+        <React.Fragment>
+          <SectionField label="Expropriation authority" required>
+            <ContactInputContainer
+              field="expropriationAuthority.contact"
+              View={ContactInputView}
+              restrictContactType={RestrictContactType.ONLY_ORGANIZATIONS}
+              displayErrorAsTooltip={false}
+            ></ContactInputContainer>
+          </SectionField>
+          <SectionField
+            label="Impacted properties"
+            required
+            tooltip="For the selected properties - corresponding property and interest details will be captured on the form."
+          >
+            <FormItem field="impactedProperties">
+              <FilePropertiesTable
+                disabledSelection={false}
+                fileProperties={acquisitionFile.fileProperties ?? []}
+                selectedFileProperties={formikProps.values.impactedProperties}
+                setSelectedFileProperties={(fileProperties: Api_PropertyFile[]) => {
+                  formikProps.setFieldValue('impactedProperties', fileProperties);
+                }}
+              ></FilePropertiesTable>
+            </FormItem>
+          </SectionField>
+          <SectionField label="Nature of interest">
+            <Input field="landInterest" />
+          </SectionField>
+          <SectionField label="Purpose of expropriation">
```

Comment:
hmm, the confluence just has "purpose" but what you have matches up with the mockup, not sure what is correct.

Reply:
Not sure either - the mockup seemed more explicit so I went with that